test/help/help-commands.test.js

```diff
@@ -4,11 +4,6 @@ const { run } = require('../utils/test-utils');
 const helpHeader = 'The build tool for modern web applications';
 
 describe('commands help', () => {
-    it('throws error if supplied as an argument for subcommands', () => {
-        const { stderr } = run(__dirname, ['serve', 'help'], false);
```

Comment:
Supplying help as an argument to any of the subcommands shouldn't be allowed.

Reply:
I think we can allow it for consistency.

`webpack --flag help` - Allowed
`webpack help --help` - Allowed
`webpack  help cmd` - Allowed

Reply:
`webpack help --help` interesting, but I think it is good

Reply:
Ah, I wanted to type `webpack help --flag`.
Just checked `webpack help --help` is allowed as well and is equivalent to `webpack help`

Reply:
/cc @jamesgeorge007

Reply:
@snitin315 Can we refactor this test to allow using `help` here

Reply:
Yes.

Reply:
updated 👍🏻